development/build/scripts.js

```diff
@@ -399,7 +399,7 @@ function createFactoredBuild({
       policyName: buildType,
       policyOverride: path.resolve(
         __dirname,
-        `../../lavamoat/browserify/${buildType}/policy-override.json`,
+        `../../lavamoat/browserify/policy-override.json`,
```

Comment:
Why the move to a single policy override?

Reply:
less work maintaining a single override file

Reply:
So I guess it doesn't matter if we override modules that exist for one build type but not others?

Reply:
correct. `lavapack` will even remove policy for packages that dont appear in the build.